espnet/nets/pytorch_backend/rnn/decoders_transducer.py

```diff
@@ -356,7 +356,7 @@ def zero_state(self, ey):
         """Initialize decoder states.
 
         Args:
-            ey (torch.Tensor): batch of input features (B, Lmax, X_dim)
+            ey (torch.Tensor): batch of input features (B, (Emb_dim + Eprojs))
```

Comment:
Sorry I was not set on the right definition.
I modified the shape of `zero_state(...)` to match call through `rnn_forward(...)` however I also call `zero_state(...)` alone with `eys` (B, Lmax, Embed_dim) or `hs_pad` (B, Tmax, Henc) in some cases to avoid a type missmatch. Second and third dim are not important here but I though it would be important to be consistent with `rnn_forward(...)`. Is that okay?

Reply:
OK